src/java.compiler/share/classes/javax/lang/model/element/AnnotationMirror.java

```diff
@@ -58,7 +58,8 @@
      * The order of the map matches the order in which the
      * values appear in the annotation's source.
      *
-     * <p>Note that an annotation mirror of a marker annotation type
+     * @apiNote
+     * An annotation mirror of a marker annotation type
      * will by definition have an empty map.
      *
      * <p>To fill in default values, use {@link
```

Comment:
Is this paragraph supposed to be part of the api notes as well?

Reply:
Yes; those are both intended to be part of the apiNote and both render as such.